django/urls/resolvers.py

```diff
@@ -329,8 +334,22 @@ def check(self):
                     id="2_0.W001",
                 )
             )
+        warnings.extend(self._check_angle_bracket_pattern())
         return warnings
 
+    def _check_angle_bracket_pattern(self):
+        match = re.match(_PATH_ANGLE_BRACKET_COMPONENT_RE, str(self._route))
+        if not match:
+            warning = Warning(
+                "Your URL pattern pattern {} has a route that contains `<` or '>', "
+                "begins with a `<` or ends with a '>', but is not matched.".format(
```

Comment:
Maybe: _"Your URL pattern pattern %s has a route that contains unmatched angle brackets '<' or '>'. This is likely a typo."_.

Reply:
U're right!